hd2605 bicycle

```diff
@@ -0,0 +1,3 @@
+Haibike
```

Comment:
good bicycle

Reply:
Tressange is a great place to to bicycle ?